tracker/track.go

```diff
@@ -61,6 +61,16 @@ func (tr *track) addPersistence() {
 	}
 }
 
+func (tr *track) addClassificationToLabel(c string) *track {
+	if tr.detClassification != nil {
+		// Find it all up to and including the date and time
+		parts := strings.Split(tr.Det.Label(), "_")
+		labelNoClass := strings.Join(parts[:4], "_")
+		return ReplaceLabel(tr, labelNoClass+"_"+c)
```

Comment:
If for some reason this does not have 4 parts, this will panic. you want to ensure that this never happens, best to check. Also, what if there is already a classification on the label? Does this check for that, either in this function, or before?